client/components/wizard-bar/docs/example.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import WizardBar from 'components/wizard-bar';
+
+module.exports = React.createClass( {
```

Comment:
Honestly, no idea - I couldn't find any docs on it in the repo :/ `WizardBar`'s `README.md` is searchable through devdocs, so it is picked up, but it's not being added to the UI Components page 🤔